src/main/groovy/com/bmuschko/gradle/docker/tasks/container/DockerCreateContainer.groovy

```diff
@@ -177,6 +177,13 @@ class DockerCreateContainer extends AbstractDockerRemoteApiTask {
     @Optional
     Long shmSize
 
+    /* 
+     * Automatically remove the container when the container's process exits.
+     * This has no effect if RestartPolicy is set.
+     */
+    @Input
+    Boolean autoRemove = false
```

Comment:
I talked myself into the wrong direction I think: I'll change the default to null and add the condition below